mock/cpp_mock_scenarios/src/move_backward/move_backward.cpp

```diff
@@ -30,10 +29,7 @@ class MoveBackwardScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit MoveBackwardScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "move_backward",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.